src/__tests__/Login.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```

Comment:
Some general comments on this test file:

1. Let's rename to `FusionAuthLoginButton.tsx` because it is testing the button itself. We should also probably create a test for the provider that tests the login call itself.

2. In general test files should be structured with all the tests/afterEach/beforeEach inside a `describe` block. We have one currently but says `urlBuilder`, it should be changed to `FusionAuthLoginButton` because that is what it is testing.